py/picca/delta_extraction/data_catalogues/desi_data.py

```diff
@@ -429,9 +428,11 @@ def read_from_minisv_desi(self, catalogue):
                                 "z": entry['Z'],
                                 "petal": entry["PETAL_LOC"],
                                 "tile": entry["TILEID"],
-                                "night": entry["NIGHT"],
+                                "night": entry[nightcol],
                                 "exposures_diff": exposures_diff,
-                                "reso": reso_in_km_per_s
+                                "reso": np.real(reso_in_km_per_s),
+                                "reso_AA": np.real(reso_in_AA),
```

Comment:
You have modified the function to return the resolution also in AA. Is there a reason for this? Do you plan to use this instead of using the resolution in km/s? If so, we should replace it accordingly, not just simply add it. Note that currently this is being ignored as it is not used by the constructor of Pk1dForest

Reply:
In the end there are at least 3 measures of resolution that might be informative here, the reso in km/s (varying with lambda) in AA (roughly constant with lambda) and the actual resolution matrix of the pixels (with masking applied). Depending on how we do P1d each of them could be used in principle. I guess it makes sense to have both available for summaries (e.g. mean reso), but the native one for DESI would be reso_AA.